.github/workflows/test-php.yml

```diff
@@ -7,7 +7,7 @@ jobs:
             WP_CORE_DIR: '/tmp/wordpress'
             COMPOSER_DEV: 1
 
-        runs-on: ubuntu-latest
+        runs-on: ubuntu-18.04
```

Comment:
Recently github have been migrating the default for ubuntu-latest to 20.04, which I think is why our tests have broken. I've pinned to 18.04 till we can migrate across. I think its just our approach with mysql that needs some work.